generators/ansible-do/templates/ansible/stack/plays/postgres.yml

```diff
@@ -0,0 +1,20 @@
+- name: Create required postgres directories on the NFS server
+  hosts: nfs_server
+  become: yes
+  tasks:
+    - name: Ensure necessary directories exist
+      file:
+        group: "nogroup"
+        owner: "nobody"
+        path: "/mnt/nfs/postgres/{{ item }}/{{ subdir }}"
+        state: directory
+        recurse: true
+      with_subelements:
+        - "{{ postgres_versions }}"
+        - subdirs
+      vars:
+        subdirs:
+          - "backups"
+          - "data"
+      when:
```

Comment:
this should preferably be at the top level
Also when NOT using nfs, postgres folders must be created on the swarm node.